frontend/packages/metal3-plugin/src/utils/host-status.ts

```diff
@@ -1,29 +1,42 @@
-import { K8sResourceKind } from '@console/internal/module/k8s';
+import { K8sResourceKind, MachineKind, NodeKind } from '@console/internal/module/k8s';
 
+import { getDeletetionTimestamp } from '@console/shared';
 import {
   getHostOperationalStatus,
   getHostProvisioningState,
   getHostErrorMessage,
-  // isNodeUnschedulable,
 } from '../selectors';
 
 import {
   HOST_STATUS_TITLES,
+  HOST_STATUS_UNDER_MAINTENANCE,
+  HOST_STATUS_STOPPING_MAINTENANCE,
   HOST_STATUS_READY,
-  // HOST_STATUS_STARTING_MAINTENANCE,
 } from '../constants';
 
-// import { NOT_HANDLED } from '../common';
+const NOT_HANDLED = null;
```

Comment:
What's the benefit of a `null` constant? Is this value expected to change in future?

Reply:
it is just a way of showing that the status resolution is handled step by step

I don't mind either way - this is just a convention from `web-ui-components`

should I change it to null?

Reply:
Yes, I'd prefer to not treat `null` as a constant, assuming it's not going to change in future.

Reply:
done